src/it/resources/expected/my_flags/python/my_flags.py

```diff
@@ -0,0 +1,17 @@
+# AUTOGENERATED FILE - DO NOT MODIFY!
+# This file generated by Djinni from my_flags.djinni
+
+from djinni.support import MultiSet # default imported in all files
+from djinni.exception import CPyException # default imported in all files
+
+from djinni import exception # this forces run of __init__.py which gives cpp option to call back into py to create exception
+
+from enum import IntEnum, unique
+
+@unique
+class MyFlags(IntEnum):
+    Flag1 = 0
+    Flag2 = 1
+    Flag3 = 2
+    No_Flags = 3
+    All_Flags = 4
```

Comment:
It should be investigated.

Reply:
Maybe because `IntFlag` was only introduced in Python 3.6 ([link](https://docs.python.org/3/library/enum.html#enum.auto)), which was released in late 2016. Do you have a minimum python3 version in mind, that should be supported?

Reply:
3.6 should be fine, earlier versions are not supported anymore, and we need to think to the future, not the past. 
3.6 will end this year, with supporting that version  we would be as much backward compatible as it makes sense.

Reply:
I just pushed a change to use `IntFlag` for Djinni flags in Python. Resulting code looks like this: https://github.com/freitass/djinni-generator/blob/93c9e4cbb952ae17558123d5b5b2515a2a152da1/src/it/resources/expected/my_flags/python/my_flags.py#L9-L16